components/form-vue/control/markdown/markdown.html

```diff
@@ -86,11 +86,12 @@
 					mode: 'gfm',
 				}"
 				/>
-		</div>
 
+		</div>
 		<app-form-control-markdown-media-items
 			v-if="showMediaItems"
 			:type="mediaItemType"
+			:disabled="disabled"
```

Comment:
I don't like doing it this way. It seems that disabled just v-ifs it out within the child component. Why not just make this a v-if here instead? `v-if="showMediaItems && !disabled"`